src/Index/Settings.php

```diff
@@ -71,16 +71,24 @@ public function __construct(BaseIndex $index)
      *
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html
      */
-    public function get(string $setting = '')
+    public function get(string $setting = '', bool $includeDefaults = false)
     {
-        $requestData = $this->request()->getData();
+        $queryParameters = [
+            'include_defaults' => $includeDefaults,
+        ];
```

Comment:
Surprised by this codecov comment as I think this part is covered in the tests. Maybe codecov is confused by the ] 🤷